src/CMakeLists.txt

```diff
@@ -108,6 +108,10 @@ endif()
 if (USE_HDFS)
     add_headers_and_sources(dbms Storages/HDFS)
     add_headers_and_sources(dbms Disks/HDFS)
+
+    if (USE_ORC AND USE_PARQUET)
+        add_headers_and_sources(dbms Storages/Hive)
+    endif()
```

Comment:
It should be a separate USE_HIVE, instead of writing `if (USE_ORC AND USE_PARQUET)`.

Reply:
Ok.